packages/core-ethereum/src/indexer/index.mock.ts

```diff
@@ -210,6 +231,12 @@ const createChainMock = (
   } as unknown as ChainWrapper
 }
 
+class TestingIndexer extends Indexer {
```

Comment:
Not sure it's a good idea to mock classes by inheriting from them, however, if it works, I'm okay with that. Just a side comment.

Reply:
The idea is to make previously private method public to call them within the test. Inheriting from the class and overwriting protected with public properties is a common pattern